packages/wrangler/package.json

```diff
@@ -60,8 +60,6 @@
     "@types/signal-exit": "^3.0.1",
     "@types/ws": "^8.2.1",
     "@types/yargs": "^17.0.7",
-    "acorn": "^8.6.0",
-    "acorn-walk": "^8.2.0",
```

Comment:
Can you run `npm i` from the root of the project to ensure that `package-lock.json` is updated.